twine/commands/__init__.py

```diff
@@ -1,3 +1,4 @@
+"""Initialize the twine.commands package."""
```

Comment:
Instead of trying to make something out of nothing here, and keeping in mind that module level docstrings don't need to start with the imperative mood, can we describe what is in the `twine.commands` package and the fact that it's contents are not a public API?